### DISCUSSION TITLE
How can I make a chat message that players can click to copy?

### DISCUSSION THREAD

**Discussion by ryleu** [Minestom/Minestom#1086] [Help] [answered]
I'm trying to make a chat message that has a room code that players can click on to copy it to their system clipboard. I know this is possible in vanilla with commands, but I don't know what `Component` to use to do it with Minestom. How can I make a copy-able chat message?

**Comment by emortaldev** [comment, marked as answer]
```java
Component.text(...).clickEvent(ClickEvent.copyToClipboard("Text To Copy"))
```

**Reply by ryleu** [reply]
Thanks, that works!
